taskcluster/ci/csharp-build/kind.yml

```diff
@@ -0,0 +1,37 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+---
+loader: glean_taskgraph.loader.build_config:loader
+
+transforms:
+  - glean_taskgraph.transforms.module_build:transforms
+  - taskgraph.transforms.job:transforms
+  - taskgraph.transforms.task:transforms
+
+job-defaults:
+  attributes:
+    run-on-pr-type: full-ci
+  run-on-tasks-for: [github-pull-request, github-push]
+  description: "{module_name} - Build and test"
+  scopes:
+    - project:releng:services/tooltool/api/download/internal
+  worker-type: b-linux
+  worker:
+    chain-of-trust: true
+    docker-image: { in-tree: csharp }
+    max-run-time: 1800
+  run:
+    pre-dotnet:
```

Comment:
There's no `pre-dotnet`.
This can just be `pre-commands` though.
It is defined here: https://github.com/mozilla/glean/blob/main/taskcluster/glean_taskgraph/job.py#L54